meta-resin-sumo/recipes-core/packagegroups/packagegroup-balena-connectivity.bbappend

```diff
@@ -3,15 +3,10 @@ CONNECTIVITY_PACKAGES_append = " crda"
 CONNECTIVITY_FIRMWARES_append = " \
     linux-firmware-bcm43143 \
     linux-firmware-iwlwifi-135-6 \
-    linux-firmware-iwlwifi-3160-7 \
-    linux-firmware-iwlwifi-3160-8 \
-    linux-firmware-iwlwifi-3160-9 \
+    linux-firmware-iwlwifi-3160 \
     linux-firmware-iwlwifi-6000-4 \
-    linux-firmware-iwlwifi-6000g2a-5 \
     linux-firmware-iwlwifi-6000g2a-6 \
-    linux-firmware-iwlwifi-6000g2b-5 \
     linux-firmware-iwlwifi-6000g2b-6 \
-    linux-firmware-iwlwifi-6050-4 \
```

Comment:
@jakogut what happened to the `iwlwifi-6000` firmware? is it included as part od `iwlwifi-3160` too?

Reply:
@alexgg The `iwlwifi-6050-4` firmware is no longer packaged upstream, only `iwlwifi-6050-5`.